packages/ui/package.json

```diff
@@ -26,11 +26,11 @@
   },
   "exports": {
     ".": {
-      "types": "./dist-all/web.d.ts",
-      "import": "./dist-all/web.js",
-      "require": "./dist-all/web.js"
+      "types": "./dist-web/web.d.ts",
```

Comment:
Trying this as `assets` aren't bring properly copied to `dist-all` so I want to check if this will work for the `.png`. Then I can try and fix the copy to `dist-all`